storage/src/main/scala/coop/rchain/storage/LMDBStore.scala

```diff
@@ -0,0 +1,284 @@
+package coop.rchain.storage
+
+import java.nio.ByteBuffer
+import java.nio.file.Path
+import java.security.MessageDigest
+
+import cats.implicits._
+import com.google.protobuf.ByteString
+import coop.rchain.storage.util._
+import org.lmdbjava.DbiFlags.MDB_CREATE
+import org.lmdbjava.{Dbi, Env, Txn}
+import coop.rchain.models.Serialize
+import coop.rchain.models.implicits.rhoInstanceWrapper
+import coop.rchain.storage.LMDBStore.{fromBB, fromBL, fromBS, hashBytes, toBB, toBL, toBS}
```

Comment:
I prefer putting this in the body of the class so that we are importing the methods from the companion object only in that scope rather than the surrounding one.

Either that or using explicit `LMDBStore.<method>` namespacing for each call.